<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 62, 80);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1ยบ Numero</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditNumero1">
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButtonCalcular">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>2ยบ Numero</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditNumero2">
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidgetTeste">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 152, 219);</string>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Fatorial</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Quadrado Perfeito</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Capicua</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Numero Perfeito</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Numero Primo</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>MMC</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>MDC</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Primos entre si</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Base 2</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Base 8</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Base 16</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </row>
      <column>
       <property name="text">
        <string>1ยบ Numero</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2ยบ Numero</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>345</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
